data-manager-api/src/main/java/org/orbisgis/datamanagerapi/datasource/IJdbcDataSource.java

```diff
@@ -98,6 +108,7 @@
      * Load a file into the database.
      *
      * @param filePath Path of the file.
+     * @return a ITable
```

Comment:
Maybe it can be replaced by 'The {@link ITable} containing the loaded data.'

Reply:
ok